experiment/autobumper/main.go

```diff
@@ -279,54 +296,75 @@ func getOncaller() (string, error) {
 	return oncall.Oncall.TestInfra, nil
 }
 
-func generatePRBody(images map[string]string) string {
+func generatePRBody(images map[string]string, enableOnCaller bool) string {
 	prowSummary := generateSummary("Prow", prowRepo, prowPrefix, true, images)
 	testImagesSummary := generateSummary("test-image", testImageRepo, testImagePrefix, false, images)
-	oncaller, err := getOncaller()
 
-	var assignment string
-	if err == nil {
-		if oncaller != "" {
-			assignment = "/cc @" + oncaller
+	var body string
+	if enableOnCaller {
```

Comment:
This is a bunch of bizarrely specific logic to have if this is supposed to be a generic tool.